### PR TITLE
yaksa: Add ROCm variant

### PR DESCRIPTION
**Pull request by raffenet**
Builds yaksa with HIP backend for AMD GPU device memory support.

### CODE REVIEW

File: var/spack/repos/builtin/packages/yaksa/package.py [hunk 1]
```diff
@@ -33,13 +33,23 @@ class Yaksa(AutotoolsPackage, CudaPackage):
     depends_on('libtool',  type='build')
     depends_on('m4',       type='build')
     depends_on('python@3:', type='build')
+    depends_on('hip', when='+rocm')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/yaksa/package.py]: I think this is already in the `ROCmPackage` base class
```suggestion
```

**Review by alalazo — CHANGES_REQUESTED** [review]
One minor comment

**Comment by raffenet** [comment]
@alalazo I committed your requested change. I think this one is good to go.

**Comment by raffenet** [comment]
@alalazo ping

**Review by alalazo — APPROVED** [review]

**Pull request merged**